### PR TITLE
Final removal of adventurelib

### PR DESCRIPTION
**Pull request by xmunoz**
- Merge Bag class
- Merge Item class
- Remove the adventurelib dependency
- Bump version

### CODE REVIEW

**Comment by xmunoz** [comment]
This fixes #7.

File: tests/whimsylib/bag_test.py [hunk 1]
```diff
@@ -68,6 +68,22 @@ def test_items_by_name(self):
...
+        book = item.Item("technological slavery")
+        woven_basket.add(cw)
+        woven_basket.add(cs)
+        woven_basket.add(book)
+
+        assert woven_basket.find("chicken wing") is cw
+        assert woven_basket.find("chapstick") is cs
+        assert woven_basket.find("CHAPSTICK") is cs
+        assert woven_basket.find("technological slavery") is book
+        assert not woven_basket.find("taxidermy racoon")
```
Comment (flosincapite) [inline tests/whimsylib/bag_test.py]: assert is None?

Comment (flosincapite) [inline tests/whimsylib/bag_test.py]: raccoon

File: whimsylib/item.py [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+
+        idle_description = kwargs.get("idle_description")
         if idle_description is None:
             idle_description = f"There is a {name} lying on the ground."
+
         self.description = description  # Fully mutable member.
         self.idle_description = idle_description  # Fully mutable member.
         self.holder = None  # Fully mutable member.
-        self._obtainable = obtainable
+        self._obtainable = kwargs.get("obtainable")
```
Comment (flosincapite) [inline whimsylib/item.py]:
```suggestion
        self._obtainable = kwargs.get("obtainable", True)
```

File: whimsylib/item.py [hunk 3]
```diff
@@ -1,25 +1,28 @@
-from adventurelib import Item as _Item
+class Item:
+    def __init__(self, name, *aliases, **kwargs):
+        """Throws ValueError if no aliases are provided."""
```
Comment (flosincapite) [inline whimsylib/item.py]: Remove this docstring; it is no longer relevant.

File: tests/whimsylib/bag_test.py [hunk 4]
```diff
@@ -68,6 +68,22 @@ def test_items_by_name(self):
 
         self.assertEqual(expected, actual)
 
+    def test_find(self):
+        """We can find items in a bag by name, case insensitively."""
+        woven_basket = bag.Bag()
```
Comment (flosincapite) [inline tests/whimsylib/bag_test.py:73]: Excellent.

**Review by flosincapite — APPROVED** [review]

**Pull request merged**